### DISCUSSION TITLE
How to remove the preinstall package?

### DISCUSSION THREAD

**Discussion by mytc** [quasarframework/quasar#9107] [General - Components / Directives / etc]
I created the quasar project with `create` quasar-cli , and upon installation I have choose to install vuex. 
How do I removed the vuex package from the project?
I am using the development version quasar (vue 3) by the way.

**Comment by hawkeye64** [comment]
@mytc I believe you can do it just be deleting the `store` folder

**Reply by mytc** [reply]
Yes. Deleting the folder works.
Previously I delete the store folder during dev mode causing the error. 
And delete the folder and re run the dev command works.
